nad_ch/config/development_local.py

```diff
@@ -68,11 +65,13 @@ def create_column_map_repository(self):
     def create_logger(self):
         return BasicLogger(__name__, logging.DEBUG)
 
-    def create_storage(self):
+    @staticmethod
```

Comment:
Does the `create_storage` method here and in the other `ApplicationContext` classes necessarily need to be static? Just curious.

Reply:
I made it static because I need to use a storage class instance in the task_queue.py file and I don't want to create an application context just for that use case. I wasn't able to pass the application context to the celery task like we discussed before due to serialization so this was a work around.